### PR TITLE
Add instructions for seeding user

### PR DESCRIPTION
**Pull request by stonefruit**
Use `BYPASS_AUTH0_LOGIN=yes` env variable to have instant login.

### CODE REVIEW

File: frontend/src/scenes/Login.js [hunk 1]
```diff
@@ -98,6 +98,11 @@ export default function Login({ locale }) {
           variant="contained"
           disableElevation
           onClick={() => {
+            // When files are served via Heroku (i.e. staging and production), the base URL and ports are the same
+            // for the backend and frontend. In these cases, the redirect to /auth/login triggers a backend endpoint
+            // and executes the oauth strategy.
+            // In development, frontend and backend use different ports (3000 and 4000), and backend endpoint is not
+            // called.
```
Comment (glencbz) [inline frontend/src/scenes/Login.js:105]: This works as expected if the call is initiated from 4000. Could you amend this to reflect that?

**Review by glencbz — CHANGES_REQUESTED** [review]
I am quite hesitant to use this approach because we can disable all of our authentication with a single misconfiguration.

In my own development, I use my personal facebook and I seed a user into the database. I wonder if there is yet another alternative we can consider.

**Comment by stonefruit** [comment]
Ah ok. Just tried logging in the way you described with Facebook. I think this is the only data that requires seeding right? I will add the sql command in the readme and remove the suggested bypass.

File: frontend/src/scenes/Login.js (hunk 1 above)
Comment (stonefruit) [reply frontend/src/scenes/Login.js]: i've put it all in the readme instead

**Comment by stonefruit** [comment]
Reduced PR to just instructions in readme.

File: README.md [hunk 2]
```diff
@@ -70,3 +103,20 @@ Just change the `ports` configuration.
...
+Specifically, the request to GET '/oauth/login' does not work when using React server (:3000)
+
+## Development using Twilio Voice
+
+Twilio requires a public URL to confirm calls via webhook. The webhook URL can be set up with https://ngrok.com/ for now.
+
+In Twilio's console, update the REQUEST URL with your ngrok `https` url.
+`Voice / TwiML / TwiML Apps / Dev backend > REQUEST URL`
+
+Note that this method only works for one developer at a time.
```
Comment (glencbz) [inline README.md:122]: We can add a new TwiML app for each dev, then we won't mess with each other's development :)

**Review by glencbz — APPROVED** [review]

**Pull request merged**